docs/Encyclopedia/pages/BWA_with_BQSR.md

```diff
@@ -0,0 +1,30 @@
+# BWA with BQSR
+
+## Description ##
+
+BWA with BQSR (Base Quality Score Recalibration) is a sequencing alignment and quality improvement workflow used in the GDC targeted sequencing data harmonization. 
+
+## Overview ##
+
+BWA with BQSR is a BWA based alignment workflow for targeted sequencing with additional step of Base Quality Score Recalibration (BQSR). BWA with BQSR is used for aligning sequence reads to a reference genome for tumor tissues only.
```

Comment:
```suggestion
BWA with BQSR is a BWA based alignment workflow for targeted sequencing with additional step of Base Quality Score Recalibration (BQSR). BWA with BQSR is used for aligning sequence reads to a reference genome.
```
Removed tumor statement - I think this was only a coincidence unless you've heard otherwise.